README.md

```diff
@@ -338,7 +338,8 @@ Table of Contents
   * [cirrus-ci.org](https://cirrus-ci.org) - Free for public GitHub repositories
   * [drone](https://cloud.drone.io/) - Drone Cloud enables developers to run Continuous Delivery pipelines across multiple architectures - including x86 and Arm (both 32 bit and 64 bit) - all in one place
   * [codemagic.io](https://codemagic.io/) - Free 500 build minutes/month
-
+  * [github.com](https://github.com/) — CI/CD with GitHub Actions (2,000 minutes per month)
```

Comment:
github is already listed several times, would rather update one of those, or even better consolidate them.  Specifically github ci/cd is here already under source code repos ```Unlimited public repositories and unlimited private repositories (with unlimited collaborators) with Pipelines for CI/CD``` so I'll close this one

thank you!